spinalcordtoolbox/scripts/sct_image.py

```diff
@@ -311,9 +312,9 @@ def main(argv=None):
     elif arguments.header is not None:
         header = im_in.header
         # Necessary because of https://github.com/nipy/nibabel/issues/480#issuecomment-239227821
-        if hasattr(im_in, "im_file"):
```

Comment:
I'm curious as to why the `hasattr` check was here to begin with. It looks like you added it in #3317, do you remember why?

Reply:
I _think_ my logic was that if `sct_image -header` was ever called on an image that wasn't loaded from a file, then the `Image` object wouldn't have the `im_in` attribute?

But, that makes no sense, since `im_in` will only ever be created from files:

https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/598aa433e19375abed4175d5b7350a9e6b5f15be/spinalcordtoolbox/scripts/sct_image.py#L206

So (prior to this PR) it would _always_ have the `im_in` attribute, making the `hasattr` check useless. 🤔